bloom-housing/bloom
# feat: add geocoding preference label

# Pull Request Template

## Issue Overview

This PR addresses #3690


## Description

Adds tag with information if we have additional fields. In reality it will check if `collectAddress` is true.

## How Can This Be Tested/Reviewed?

On partners page go to `add/edit` listing. `Housing Preferences` table should have additional column. 
Same with table after click `add preference`, `edit preference`.
In `select preferences` there will be tag below preference, and additional text near option after click `preview`.
Make sure that `programs` section is not affected by that change.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
